secret-hitler-online-interface/src/App.js

```diff
@@ -431,11 +431,12 @@ class App extends Component {
     };
 
     shouldJoinButtonBeEnabled() {
-        return (this.state.joinLobby.length === LOBBY_CODE_LENGTH) && (this.state.joinName.length !== 0)
```

Comment:
See the similar note below. Currently, the Join button is always disabled, even when putting in a name and lobby code.